mk/k3d.mk

```diff
@@ -106,7 +106,7 @@ k3d/cleanup-docker-credentials:
 	@rm -f /tmp/.kuma-dev/k3d-registry.yaml
 
 .PHONY: k3d/start
-k3d/start: ${KIND_KUBECONFIG_DIR} k3d/network/create k3d/setup-docker-credentials
+k3d/start: ${KIND_KUBECONFIG_DIR} k3d/network/create k3d/setup-docker-credentials \
```

Comment:
I don't understand how this helps... this is a list of target prerequisites, with `\` what is it?

Reply:
I agree. Also seeing the weirdness of the fix I think this warrants a lengthier PR description to explain how a magical `\` solves this problem

Reply:
When it's using `calico`, we need to add another step as dependency which is defined in the next line.
it was good previously and we had a backport PR broke the declaration here yesterday. So here is to bring back the continuation char that was missing from that PR.
https://github.com/kumahq/kuma/pull/12277/files#diff-6a9a6d7b0c33e516a1579d4ea2ccdfdbec5bd689d4f15bcab302e2a80b0d7ee4R109

Reply:
I didn't notice this. @jijiechen Thanks for spotting it

Reply:
Cool added all this info in the PR description